src/net.h

```diff
@@ -506,6 +506,8 @@ class CNodeStats
     double dMinPing;
     std::string addrLocal;
     CAddress addr;
+    // Bind address that this connection came in on (for incoming connections)
```

Comment:
I think it would be helpful if you could add comments for `addr` and `addrLocal` above, they contain "ip address and port of the peer" and "local address" according to RPC documention, and say specifically how `addrListen` is different from `addrLocal`.